openqabot/types/incidents.py

```diff
@@ -109,8 +110,6 @@ def __call__(
                     full_post["openqa"]["VERSION"] = self.settings["VERSION"]
                     full_post["qem"]["version"] = self.settings["VERSION"]
                     full_post["openqa"]["DISTRI"] = self.settings["DISTRI"]
-                    full_post["openqa"]["_ONLY_OBSOLETE_SAME_BUILD"] = "1"
```

Comment:
My browser might be misbehaving, but I still see this line as removed. Is that correct?

Reply:
I trust your browser more than me :)

Reply:
I trust your browser more than me :)